frontend/scripts/react-components/dashboard-element/dashboard-element.container.jsx

```diff
@@ -27,9 +29,16 @@ class DashboardElementContainer extends React.Component {
   state = {
     modalOpen: true,
     goBackOnCloseModal: true,
-    step: DashboardElement.steps.WELCOME
+    step: hasVisitedBefore ? DashboardElement.steps.PANEL : DashboardElement.steps.WELCOME
   };
 
+  componentDidMount() {
+    if (!hasVisitedBefore) {
+      hasVisitedBefore = true;
```

Comment:
The reason you need to set the hasVisitedBefore to true is because this value is read on `dashboard-element.container.jsx` import and not on instantiation. This is because the value is initialised out of the class. If you move the hasVisitedBefore into the class as a class property (just like `state`) then there's no need to update the value to true because the value will be read everytime the component is instantiated.